Sources/CardPayments/CardClient.swift

```diff
@@ -45,7 +45,7 @@ public class CardClient: NSObject {
                 let result = try await vaultAPI.updateSetupToken(cardVaultRequest: vaultRequest).updateVaultSetupToken
                 
                 // TODO: handle 3DS contingency with helios link & add unit tests
-                    if let link = result.links.first(where: { $0.rel == "approve" && $0.href.contains("helios") }) {
+                if let link = result.links.first(where: { $0.rel == "approve" && $0.href.contains("helios") }) {
                     let url = link.href
                     print("3DS url \(url)")
```

Comment:
I believe we are waiting to display the link in an ASWebAuthSession until the url scheme redirect is working? If so, maybe we can update the TODO note to call that out.

Reply:
Sure, this change was just part of fixing the indentation, we already have a draft where we are addressing this https://github.com/paypal/paypal-ios/pull/238, we will gladly resolve your comments in that PR as well.